src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -443,6 +462,16 @@ export class MgtPersonCard extends MgtTemplatedComponent {
    * @memberof MgtPersonCard
    */
   protected async loadState() {
+    const provider = Providers.globalProvider;
+
+    if (!provider || provider.state !== ProviderState.SignedIn) {
+      return;
+    }
+
+    if (this.personDetails) {
```

Comment:
Make sure any change in `userId`, `personQuery`, or `inheritDetails` invalidates the current `personDetails`. For reference, check out the `attributeChangedCallback` function of `mgt-person`.

Reply:
Done